Cargo.lock

```diff
@@ -539,18 +552,38 @@ version = "0.10.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fabbf1ead8a5bcbc20f5f8b939ee3f5b0f6f281b6ad3468b84656b658b455259"
 dependencies = [
- "phf_macros",
- "phf_shared",
+ "phf_macros 0.10.0",
+ "phf_shared 0.10.0",
```

Comment:
Interesting.

Reply:
Opened a PR in swc. This is not a big deal and can be fixed in the next upgrade.